tests/tests/database_api_tests.cpp

```diff
@@ -1679,5 +1679,309 @@ BOOST_AUTO_TEST_CASE( asset_in_collateral )
+
+   } catch (fc::exception& e) {
+      edump((e.to_detail_string()));
+      throw;
+   }
+}
+BOOST_AUTO_TEST_CASE( api_limit_get_full_accounts2 ) {
+
+   try {
+      graphene::app::database_api db_api(db, &(this->app.get_options()));
+      vector<string> accounts;
+      for (int i=0; i<201; i++) {
+         std::string acct_name = "mytempacct" + std::to_string(i);
+         const account_object& account_name=create_account(acct_name);
+         accounts.push_back(account_name.name);
+      }
+      GRAPHENE_CHECK_THROW(db_api.get_full_accounts(accounts, false), fc::exception);
+      accounts.erase(accounts.begin());
+      auto full_accounts = db_api.get_full_accounts(accounts, false);
+      BOOST_REQUIRE_EQUAL(full_accounts.size(), 200u);
+   } catch (fc::exception& e) {
+      edump((e.to_detail_string()));
+      throw;
+   }
+}
+BOOST_AUTO_TEST_CASE(api_limit_get_withdraw_permissions_by_recipient){
+   try{
+      graphene::app::database_api db_api( db, &app.get_options());
+      ACTORS((bob)) ;
+      withdraw_permission_id_type withdraw_permission;
+      transfer(account_id_type(), bob_id, asset(100));
+      GRAPHENE_CHECK_THROW(db_api.get_withdraw_permissions_by_recipient(
+         "bob",withdraw_permission, 251), fc::exception);
+      vector<withdraw_permission_object> result =db_api.get_withdraw_permissions_by_recipient(
+         "bob",withdraw_permission,250);
+      BOOST_REQUIRE_EQUAL( result.size(), 0u);
+   }catch (fc::exception& e) {
+      edump((e.to_detail_string()));
+      throw;
+   }
+}
+BOOST_AUTO_TEST_CASE(api_limit_get_withdraw_permissions_by_giver){
+   try{
+      graphene::app::database_api db_api( db, &app.get_options());
+      ACTORS((bob)) ;
+      withdraw_permission_id_type withdraw_permission;
+      transfer(account_id_type(), bob_id, asset(100));
+      GRAPHENE_CHECK_THROW(db_api.get_withdraw_permissions_by_giver(
+         "bob",withdraw_permission, 251), fc::exception);
+      vector<withdraw_permission_object> result =db_api.get_withdraw_permissions_by_giver(
+         "bob",withdraw_permission,250);
+      BOOST_REQUIRE_EQUAL( result.size(), 0u);
+   }catch (fc::exception& e) {
+      edump((e.to_detail_string()));
+      throw;
+   }
+}
+BOOST_AUTO_TEST_CASE(api_limit_get_trade_history_by_sequence){
+   try{
+      app.enable_plugin("market_history");
+      graphene::app::application_options opt=app.get_options();
+      opt.has_market_history_plugin = true;
+      graphene::app::database_api db_api( db, &opt);
+      ACTORS((bob) (alice) (fred)) ;
+      const auto& bitusd = create_bitasset("USDBIT", bob_id);
+      asset_id_type asset_1, asset_2;
+      asset_1 = bitusd.id;
+      asset_2 = asset_id_type();
+      GRAPHENE_CHECK_THROW(db_api.get_trade_history_by_sequence(
+         std::string( static_cast<object_id_type>(asset_1)),
+         std::string( static_cast<object_id_type>(asset_2)),
+         0,fc::time_point_sec(1532008940), 251), fc::exception);
```

Comment:
That arbitrary timestamp looks confusing. Perhaps just use `fc::time_point_sec()` instead?

Reply:
Done the chnage